routes/facetsWithArticles.js

```diff
@@ -13,6 +13,57 @@ router.get('/test', async (req, res, next) => {
 	res.render("facetsWithArticles/test", { facetsJson: JSON.stringify(results) } );
 });
 
+router.get('/clusteredImages/:template/:facet', async (req, res, next) => {
+
+	const results = await article.getArticlesInTopics(1, req.params.facet);
+
+	switch(req.params.template){
```

Comment:
Not quite what I had in mind. More: ```view = `facetsWitArticles/clusteredImages/${req.params.template}` ```
Saves a big switch statement. Similarly `data = results.breakdown.splice(0,1)`; and `if( req.params.template.startsWith('five') || req.params.templates === 'six) { data = results.breakdown.splice(0,1) }`

Reply:
Ah, ok.
Done